ArduCopter/Copter.cpp

```diff
@@ -309,6 +309,21 @@ bool Copter::set_target_velocity_NED(const Vector3f& vel_ned)
     return true;
 }
 
+bool Copter::set_target_angle_and_climbrate(float roll_deg, float pitch_deg, float yaw_deg, float climb_rate_cms, bool use_yaw_rate, float yaw_rate_rads)
```

Comment:
sorry to keep drip feeding changes but I suspect that we should make the climb rate in m/s instead of cm/s.

Reply:
yaw rate in deg per second also? to match roll, pitch and yaw

Reply:
Yes, that makes sense too.  good catch.

Reply:
fixed units